src/main/resources/config/application-dev.yml

```diff
@@ -75,7 +75,7 @@ spring:
             discovery:
                 enabled: true
     kafka:
-        bootstrap-servers: localhost:9093
```

Comment:
why port changed?

Reply:
because default kafka port is 9092

Reply:
in all our MS default value - 9093
i think such changes must be done for whole services after conversation
for some reason jhipster generate exactly 9093